DataCert/mkAndSubmitJobs.py

```diff
@@ -12,16 +12,19 @@
 parser.add_argument("--pkldir", type=str, default="../brildata", help="Path to BRIL pickle files.")
 parser.add_argument("-q", "--queue", type=str, default="8nh", help="lxbatch queue (default:  8nh)")
 parser.add_argument('-v', '--includeVertices', default=True, action="store_false", help="Include vertex counting")
-
+parser.add_argument('-o', '--outPath', default="", help="Specify the path of the output files")
 args=parser.parse_args()
 
 
 def MakeJob(outputdir,jobid,filename,minfill):
     joblines=[]
     joblines.append("source /cvmfs/cms.cern.ch/cmsset_default.sh")
-    joblines.append("cd "+outputdir)
+    joblines.append("cd /afs/cern.ch/work/j/jingyu/CMSSW_7_4_7/src")
```

Comment:
This line will only work for you.  In general you should not put a path to your private area in a script that other people will use.